docs/user/twrp.md

```diff
@@ -36,6 +36,10 @@ There are many ways to install TWRP, but here's one that works well.
 
         $ fastboot flash recovery <twrp filename>.img
 
+    If above command failed with no `recovery` partition error, you can try the following command:
```

Comment:
It looks like the official instructions suggest a two-step process for installing TWRP permanently on devices without the `recovery` partition: https://twrp.me/google/googlepixelxl.html. First `fastboot boot twrp.img` and then install a new TWRP zip installer file. But if this command works for now, I think we can keep it to keep the instructions simple.

Reply:
It works fine for me with sailfish.